integration/integration_test.go

```diff
@@ -0,0 +1,85 @@
+package integration
+
+import (
+	"flag"
+	"math/rand"
+	"net"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/sirupsen/logrus"
+
+	mapp "github.com/grafana/prometheus-pulsar-remote-write/pkg/app"
+)
+
+type Clocker interface {
+	Now() time.Time
+}
+
+var clock Clocker = &testClock{}
+
+type testClock struct {
+	t time.Time
+}
+
+var app = mapp.New()
```

Comment:
Can you move this to after the `func (c *testClock) Now() time.Time` function? This `app` is not used in `testClock`.

Reply:
Put it close to `TestMain` now